src/main/java/practice/StreamPractice.java

```diff
@@ -50,7 +73,12 @@ public List<Person> selectMenByAge(List<Person> peopleList, int fromAge, int toA
      */
     public List<Person> getWorkablePeople(int fromAge, int femaleToAge,
                                           int maleToAge, List<Person> peopleList) {
-        return Collections.emptyList();
+        Predicate<Person> predicate = p -> p.getAge() >= fromAge
```

Comment:
Remember about appropriate naming.
```suggestion
        Predicate<Person> isWorkablePredicate = p -> p.getAge() >= fromAge
```